CHANGES/3254.misc

```diff
@@ -0,0 +1 @@
+Add support for reading zstd metadata.
```

Comment:
i'd move this to .feature

Reply:
The automation will get angry at us for back porting in that case, which we might not be able to avoid.  Fedora will likely switch over their repos in the near-ish future (a few months), and then EPEL.

Reply:
gotcha, in that case this is correct.